### PR TITLE
Fixed Issue #1 Issue #6 and Added Feature 1 and Feature 2 and Feature 3 also fixed same symbol error

### PR DESCRIPTION
**Pull request by geekymanas**

### CODE REVIEW

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 1]
```diff
@@ -26,14 +26,21 @@
...
+    private Integer count = 0;
+    JPanel playingArea = new JPanel();
     private void reset(){
         for(int i = 0; i < 3; i++){
             for(int j = 0; j < 3; j++){
                 playingLabels[i][j].setText(".");
             }
         }
         currentPlayer = 0;
+        jLabel3.setText("");
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java:42]: Use meaningful variable names throughout the code.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 2]
```diff
@@ -91,19 +99,36 @@ private void check(int currentPlayer){
...
     
     private void cellClicked(MouseEvent evt){
+        boolean player = false;
         JLabel currentCell = (JLabel) evt.getComponent();
+        if(!(".".equals(currentCell.getText()))){
+                JOptionPane.showMessageDialog(this,"This Position Already Chosen");
+            }
+        else{
+            count++;
         if(currentPlayer == 0){
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java:117]: indent the code properly.

**Comment by urdarinda** [comment]
@nikilagar  I think we should close this PR as the commits and the title do not match to the guidelines.

**Comment by urdarinda** [comment]
@geekymanas You are supposed to make a new branch and create a PR according to the guidelines.

**Comment by geekymanas** [comment]
Where are the guidelines provided for reference

**Review by nikilagar — CHANGES_REQUESTED** [review]

**Comment by nikilagar** [comment]
See this is a demo so that we can see what problem could occur... In the
next post we'll be briefly describing all about it...

**Pull request closed without merging**